plugins/Identify.jsx

```diff
@@ -99,7 +99,12 @@ class Identify extends React.Component {
             let queryableLayers = [];
             queryableLayers = IdentifyUtils.getQueryLayers(this.props.layers, this.props.map);
             queryableLayers.forEach(l => {
-                const request = IdentifyUtils.buildRequest(l, l.queryLayers.join(","), clickPoint, this.props.map, this.props.params);
+                const params = {...this.props.params};
+                // Apply layer filters to identify request
+                if (l.params.FILTER) {
+                    params.filter = l.params.FILTER
+                }
+                const request = IdentifyUtils.buildRequest(l, l.queryLayers.join(","), clickPoint, this.props.map, params);
```

Comment:
Perhaps better to apply the layer filters inside IdentifyUtils.buildRequest?

Reply:
Yes, but as I said identifyRegion uses the param `FILTER_GEOM` and if I add the param `filter` it doesn't filter properly, it returns features from outside the identified region.
I don't really know what is happening, to me it seems like a qgis server thing, no?

Reply:
Yes I believe it is a QGIS Server limitation

Reply:
Okay, well, I will still move it inside IdentifyUtils.buildRequest as I see that it's only called for identifyPoint

Reply:
I believe the cause is this: https://github.com/qgis/QGIS/blob/master/src/server/services/wms/qgswmsrenderer.cpp#L1407